packages/content-management/table_list_view_table/src/table_list_view_table.tsx

```diff
@@ -660,7 +659,7 @@ function TableListViewTableComp<T extends UserContentCommonSchema>({
         name: i18n.translate('contentManagement.tableList.listing.table.actionTitle', {
           defaultMessage: 'Actions',
         }),
-        width: '100px',
+        width: `${32 * actions.length}px`,
```

Comment:
I wanted to adjust a little bit the columns. Make it a bit tidier with new creator column and have less white space for actions and updated by